### DISCUSSION TITLE
Does support streaming / piping ?

### DISCUSSION THREAD

**Discussion by georgecoca** [spatie/laravel-backup#1110] [Q&A] [answered]
Hi,

I was wondering if laravel-backup supports streaming the dump directly to destination, eg: s3. There may be situations where you do not have enough space on the server and want to pipe the output directly instead of storing into into a temp folder.

Thanks,

**Comment by freekmurze** [comment, marked as answer]
No this is not supported. It would be an awesome feature, but I'm not sure how to do this.

If you know a good way to do this, I'd welcome a PR.

**Comment by georgecoca** [comment]
Thanks for your reply. Well, I never did this in PHP but I've accomplished this as a stand alone script using Node. If I get some free time, I would love to implement this into PHP.
